src/Core/ServiceWrapper/app.manifest

```diff
@@ -0,0 +1,10 @@
+﻿<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="WinSW" type="win32" />
+  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v2">
+    <security>
+      <requestedPrivileges xmlns="urn:schemas-microsoft-com:asm.v3">
+        <requestedExecutionLevel level="requireAdministrator" uiAccess="false" />
```

Comment:
uiAccess is `false` for the service mode, but it is an option for the CLI mode

Reply:
IIUC we'll need a digital signature to enable `uiAccess`.

Reply:
What does `uiAccess` do?

Reply:
Could you please explain why we need UIAccess? I've got a different direction for this.